lib/seattleflu/id3c/cli/command/etl/redcap_det_fh_airs.py

```diff
@@ -0,0 +1,910 @@
+"""
+Process DETs for the Fred Hutch AIRS study into the FHIR receiving table
+"""
+import logging
+from dateutil.relativedelta import relativedelta
+from enum import Enum
+from id3c.cli.command.etl import redcap_det
+from id3c.cli.redcap import is_complete, Record as REDCapRecord
+from id3c.db import find_identifier
+from seattleflu.id3c.cli.command import age_ceiling
+from .redcap import *
+from .redcap_map import *
+
+LOG = logging.getLogger(__name__)
+
+
+class AIRSProject():
+    id: int
+    lang: str
+    command_name: str
+
+    def __init__(self, project_id: int, lang: str, command_name: str) -> None:
+        self.id = project_id
+        self.lang = lang
+        self.command_name = command_name
+
+
+REDCAP_URL  = 'https://redcap.fredhutch.org/'     # REDCap server hostname
+REDCAP_PID  = 1372                                # REDCap project ID
+REDCAP_LANG = 'en'                                # Survey language
+ETL_COMMAND = 'fh-airs'                           # Run as 'id3c etl redcap-det <ETL_COMMAND>'
+
+PROJECTS = [
+        AIRSProject(REDCAP_PID, REDCAP_LANG, ETL_COMMAND)
+    ]
+
+LANGUAGE_CODE = {
+project.id: project.lang
+    for project in PROJECTS }
+
+class CollectionMethod(Enum):
+    SWAB_AND_SEND = 'swab_and_send'
+    KIOSK = 'kiosk'
+    UW_DROPBOX = 'uw_dropbox'
+
+class EventType(Enum):
+    ENROLLMENT = 'enrollment'
+    ENCOUNTER = 'encounter'
+
+REVISION = 1
+
+INTERNAL_SYSTEM = "https://seattleflu.org"
+# TODO: ENCOUNTER_EVENT_NAME, SWAB_AND_SEND_SITE, UW_DROPBOX_SITE
+# may not be necessary for this study
+ENROLLMENT_EVENT_NAME = "screening_and_enro_arm_1"
+ENCOUNTER_EVENT_NAMES = [
+    "week_01_arm_1",
+    "week_02_arm_1",
+    "week_03_arm_1",
+    "week_04_arm_1",
+    "week_05_arm_1",
+    "week_06_arm_1",
+    "week_07_arm_1",
+    "week_08_arm_1",
+    "week_09_arm_1",
+    "week_10_arm_1",
+    "week_11_arm_1",
+    "week_11_arm_1",
+    "week_12_arm_1",
+    "week_13_arm_1",
+    "week_14_arm_1",
+    "week_15_arm_1",
+    "week_16_arm_1",
+    "week_17_arm_1",
+    "week_18_arm_1",
+    "week_19_arm_1",
+    "week_20_arm_1",
+    "week_21_arm_1",
+    "week_22_arm_1",
+    "week_23_arm_1",
+    "week_24_arm_1",
+    "week_25_arm_1",
+    "week_26_arm_1",
+]
+
+SWAB_AND_SEND_SITE = 'AIRSSwabNSend'
```

Comment:
We'll want to map this new site in the warehouse.sites table. That doesn't necessarily need to be handled as part of this PR, but maybe once this is finalized, can you please add a trello task to add the site to schema/deploy/warehouse/site/data.sql  ? Thanks!

Reply:
That's done: https://trello.com/c/aEd5FD0U/1381-add-airsswabnsend-site-to-warehousesite-table